crates/teloxide-core/src/types/message.rs

```diff
@@ -40,6 +40,11 @@ pub struct Message {
     /// forwarded to the discussion group
     pub sender_chat: Option<Chat>,
 
+    /// If the sender of the message boosted the chat, the number of boosts
+    /// added by the user
+    // Yes, it's i32 ¯\_(ツ)_/¯: https://github.com/tdlib/td/blob/cb164927417f22811c74cd8678ed4a5ab7cb80ba/td/telegram/MessagesManager.h#L1004
+    pub sender_boost_count: Option<i32>,
```

Comment:
I think, it should have the same type as `ChatBoostAdded::boost_count` (could a person make > 255 boosts..hm?)

At least `count` shouldn't be negative.

Reply:
I think you can get more than 255 boosts because each premium gift gives 3 boosts to the giver. 4 boosts from Premium + 84 gifts = 256 boosts. It will be a bit expensive, but still possible. To exceed the u16 limit, you'd need to gift premium 21844 times, and that seems impossible.

Reply:
Yeah, it's like saving on matches, `u16` will be good. Just want to make sure, that we use the same type for similar entities